.github/workflows/unit-tests.yml

```diff
@@ -13,6 +13,20 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
+    services:
+      postgres:
+        image: ankane/pgvector:v0.5.0
```

Comment:
Should we use a base postgres image and then use the initialization functions to set up pgvector?

Reply:
That's a reasonable option, but installing PGVector is [non-trivial](https://github.com/pgvector/pgvector#installation) (as in, it doesn't ship out of the box with Postgres and we'd need to orchestrate compiling/installing it). This image handles that step for us, but we still have to enable the extension.

Reply:
Ah I didn't realize that this is the [official image](https://github.com/pgvector/pgvector#docker), let's just stick with this then 👍